04_removeFromArray/removeFromArray.js

```diff
@@ -1,5 +1,5 @@
-const removeFromArray = function() {
-
+const removeFromArray = function(...arg) {
+    
```

Comment:
Reverting unrelated file to original state
```suggestion
const removeFromArray = function() {

```